app/models/person.rb

```diff
@@ -100,6 +100,14 @@ def agent_projects
     Project.where("primary_agent_id = ? OR secondary_agent_id = ?", id, id)
   end
 
+  def active_agent_projects
+    # Status values can be found at Loan.status_option_set.options and
+    # BasicProject.status_option_set.options
+    active_statuses = ["active", "changed", "possible", "prospective", "refinanced", "relationship",
+      "relationship_active"]
```

Comment:
This should perhaps be a constant on `Project`? Also please switch to `%w(...)` notation, it will fit on one line then.